images/entrypoint.sh

```diff
@@ -317,7 +317,7 @@ if [ "$MULTUS_CONF_FILE" == "auto" ]; then
 import json,sys
 conf = json.load(sys.stdin)
 capabilities = {}
-for capa in [p['capabilities'] for p in conf['plugins'] if 'capabilities' in p]:
+for capa in [p['capabilities'] for p in conf.get('plugins', []) if 'capabilities' in p]:
```

Comment:
So, to my understanding this bit of code is used to get the list of capabilities the master plugin supports.

Shouldn't we support the case where the master plugin is not specified as a configuration list ?

Reply:
Updated to cover master plugin